AmahiAnywhere/AmahiAnywhere/Services/Network.swift

```diff
@@ -0,0 +1,459 @@
+//
+//  Network.swift
+//  AmahiAnywhere
+//
+//  Created by Jogendra Singh on 14/02/18.
+//  Copyright © 2018 Amahi. All rights reserved.
+//
+
+import UIKit
+import Foundation
+import Alamofire
+
+fileprivate struct Defaults {
+  static let timeoutInternval: TimeInterval = 20
+}
+
+class Network {
+
+  static let shared = Network()
+
```

Comment:
Network class looks good. I think it is an overkill with what we have on the app at the moment. We can always add other implementations when we are using them as the put and patch. 
The .get and .post method are useful at the moment. I think whoever is implementing other methods can add them to suit the way the API implements these methods. 

Also, the network indicator should be managed in the presenter class making the network call. The UIViewController has already been extended so you can easily show and dismiss the loading dialog.  You should check out the base package and check what is there. There is also an API endpoint class for managing ApiEndpoints easily on the app.

Reply:
Hi @codedentwickler Thank you so much for reviewing :)
I added this class keeping common API calling methods in mind.  I am also agree with your points too.